Id Id
string username
string name
string university
